src/App.js

```diff
@@ -14,11 +14,57 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selected: 'Jam',
+  }
+
+  render() {
+    const { selected } = this.state;
+
+    return (
+      <>
+        {selected
+          ? (
+            <h1>
+              Selected good: -
+              {selected}
+
+              <button
+                type="button"
+                onClick={() => {
+                  this.setState({ selected: null });
+                }}
+              >
+                X
+              </button>
+            </h1>
+          ) : (
+            <h1>
+              No goods selected
+            </h1>
+          )}
+
+        {goodsFromServer.map(el => (
+          <ul key={el}>
+            <li>
```

Comment:
```suggestion
        <ul>
          {goodsFromServer.map(el => (
            <li key={el}>
```
Now you are creating a lot of lists with one item. It's wrong. You should create one list (`ul`) with many list items (`li`). Also, key shoul be used for `li` not `ul`